### ISSUE TITLE
Get-PASSafeMember with includePredefinedUsers eq True and more than 26 members in the safe.

### ISSUE THREAD

**Issue by InconstantRO** [pspete/psPAS#433]
**Describe the issue**
Get-PASSafeMember with includePredefinedUsers eq True and more than 26 members in the safe throws "[500] String was not recognized as a valid Boolean." error. Error is thrown by Invoke-PASRestMethod method in line 295 (in Get-PASSafeMember.ps1 file). It also returns first 25 members, but fails to query next.

**To Reproduce**
1. Assign 26+ members (35 to be sure) to your PVWAReports safe.
2. Logon to PAM and run next command to get safe PVWAReports members, including predefined users:
Get-PASSafeMember -safename "PVWAReports" -includePredefinedUsers $true

**Expected behavior**
Expected to get all safe members.

**Screenshots & Console Output**
If applicable, add screenshots and/or console output to help explain your problem.

```powershell

<--
[500] String was not recognized as a valid Boolean.
-->

```

**Your Environment**
Include relevant details about your environment

* PowerShell Version: PSVersion 5.1.19041.1682
* psPAS Version: 5.4.94
* CyberArk Version: 12.2.4.76

**Additional context**
I think the problem is in double question mark in resulting URL (?filter, ?limit):
https://myserver.com/PasswordVault/api/Safes/PVWAReports/Members?filter=includePredefinedUsers%20eq%20True?limit=25&OffSet=25

**Fix**
To fix the problem this line should be changed from:
```powershell

$Null = $Members.AddRange((Invoke-PASRestMethod -Uri "$URI`?limit=$Limit&OffSet=$Offset" -Method GET -WebSession $Script:WebSession -TimeoutSec $TimeoutSec).value)

```
to
```powershell

if ($URI -match ".*\?.*") {
	$Null = $Members.AddRange((Invoke-PASRestMethod -Uri "$URI`&limit=$Limit&OffSet=$Offset" -Method GET -WebSession $Script:WebSession -TimeoutSec $TimeoutSec).value)
} else {
	$Null = $Members.AddRange((Invoke-PASRestMethod -Uri "$URI`?limit=$Limit&OffSet=$Offset" -Method GET -WebSession $Script:WebSession -TimeoutSec $TimeoutSec).value)
}

```

**Comment by pspete** [comment]
Hi @InconstantRO - many many thanks for reporting this.
Confirmed the observed behaviour, fix developed and now present in the Dev branch.

**Comment by pspete** [comment]
Fix now published in version 5.4.101.
Also added unit tests to check the URL is getting formed as expected.
Thanks again for the report.
